tla/consensus/ccfraft.tla

```diff
@@ -82,6 +82,9 @@ CONSTANTS
 CONSTANTS Servers
 ASSUME Servers /= {} /\ IsFiniteSet(Servers)
 
+\* Initial term used by the Start node in the network
+CONSTANT StartTerm
```

Comment:
The config definitions won't be necessary with `StartTerm == 2`.

Reply:
Will fix